dev/how-to/deploy/orchestrated/tidb-in-kubernetes/alibaba-cloud.md

```diff
@@ -183,36 +181,164 @@ kubectl get pods --namespace tidb -o wide --watch
+| `monitor_instance_type` | The instance type of monitoring components | `ecs.c5.xlarge` |
+| `override_values` | The `values.yaml` configuration file of the TiDB cluster. You can read it using the `file()` function | `nil` |
+| `local_exec_interpreter` | The interpreter that executes the command line instruction | `["/bin/sh", "-c"]` |
+
+## Manage multiple Kubernetes clusters
+
+It is recommended to use a separate Terraform module to manage a specific Kubernetes cluster. (A Terraform module is a directory that contains the `.tf` script.)
+
+`deploy/aliyun` combines multiple reusable Terraform scripts in `deploy/modules`. To manage multiple clusters, perform the following operations in the root directory of the `tidb-operator` project:
+
+1. Create a directory for each cluster. For example:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    mkdir -p deploy/aliyun/aliyun-staging
+    ```
+
+2. Refer to `main.tf` in `deploy/aliyun` and write your own script. For example:
+
+    ```hcl
+    provider "alicloud" {
+        region     = <YOUR_REGION>
+        access_key = <YOUR_ACCESS_KEY>
+        secret_key = <YOUR_SECRET_KEY>
+    }
+    
+    module "tidb-operator" {
+        source     = "../modules/aliyun/tidb-operator"
+        
+        region          = <YOUR_REGION>
+        access_key      = <YOUR_ACCESS_KEY>
+        secret_key      = <YOUR_SECRET_KEY>
+        cluster_name    = "example-cluster"
+        key_file        = "ssh-key.pem"
+        kubeconfig_file = "kubeconfig"
+    }
+    
+    provider "helm" {
+        alias    = "default"
+        insecure = true
+        install_tiller = false
+        kubernetes {
+            config_path = module.tidb-operator.kubeconfig_filename
+        }
+    }
+    
+    module "tidb-cluster" {
+        source = "../modules/aliyun/tidb-cluster"
+        providers = {
+            helm = helm.default
+        }
+        
+        cluster_name = "example-cluster"
+        ack          = module.tidb-operator
+    }
+    
+    module "bastion" {
+        source = "../modules/aliyun/bastion"
+        
+        bastion_name             = "example-bastion"
+        key_name                 = module.tidb-operator.key_name
+        vpc_id                   = module.tidb-operator.vpc_id
+        vswitch_id               = module.tidb-operator.vswitch_ids[0]
+        enable_ssh_to_worker     = true
+        worker_security_group_id = module.tidb-operator.security_group_id
+    }
+    ```
+
+You can customize this script. For example, you can remove the `module "bastion"` declaration if you do not need the bastion machine.
+
+> **Note:**
+>
+> You can copy the `deploy/aliyun` directory. But you cannot copy a directory on which the `terraform apply` operation is currently performed. In this case, it is recommended to clone the repository again and then copy it.
 
 ## Destroy
 
-It may take some while to finish destroying the cluster.
+It may take a long time to finish destroying the cluster.
```

Comment:
```suggestion
It may take a while to finish destroying the cluster.
```
I prefer not to use "take a long time" here because it doesn't sound good to me before I do something.